example_1/bringup/launch/rrbot.launch.py

```diff
@@ -17,109 +17,108 @@
 from launch.actions import DeclareLaunchArgument, RegisterEventHandler
 from launch.conditions import IfCondition
 from launch.event_handlers import OnProcessExit
-from launch.substitutions import Command, FindExecutable, LaunchConfiguration, PathJoinSubstitution
+from launch.substitutions import Command, LaunchConfiguration, PathJoinSubstitution
```

Comment:
The missing import to fix the failing tests:
```suggestion
from launch.substitutions import Command, LaunchConfiguration, PathJoinSubstitution, PathSubstitution
```
But I'd prefer to use either PathJoinSubstitution or PathSubstitution (which one is a matter of personal preference, you can decide), not both in one file.